config/environment.rb

```diff
@@ -3,3 +3,14 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionMailer::Base.smtp_settings = {
+  :user_name => 'raoahmar87@gmail.com', # This is the string literal 'apikey', NOT the ID of your API key
+  :password => 'potvyylwinfgoysp', # This is the secret sendgrid API key which was issued during API key creation
+  :domain => 'yourdomain.com',
+  :address => 'smtp.gmail.com',
+  :port => 587,
+  :authentication => :plain,
+  :enable_starttls_auto => true
```

Comment:
ActionMailer::Base.smtp_settings = {
  user_name: 'raoahmar87@gmail.com',
  password: 'potvyylwinfgoysp',
  domain: 'yourdomain.com',
  address: 'smtp.gmail.com',
  port: 587,
  authentication: :plain,
  enable_starttls_auto: true
}

Reply:
move to development.rb